### PR TITLE
SRCH-3168 update PR template

### PR DESCRIPTION
**Pull request by amanifarooque**
## Summary
- Updating the PR template per team discussions
- I'm _not_ running a bundle update as part of these changes; if we want to do that, we can assign a dev to these changes.
- Template below reflects the proposed changes (manually pasted in for now)
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji.
 
#### Functionality Checks

- [x] Automated checks pass. If CodeClimate checks do not pass, explain reason for failures:

 ⚠️  You have run `bundle update` and committed your changes to Gemfile.lock. - **N/A - not running bundle update** 
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] You have squashed your commits into a single commit (exceptions: your PR includes commits with formatting-only changes, such as required by Rubocop or Cookstyle, or if this is a feature branch that includes multiple commits)
 
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket
 
#### Process Checks

- [x] You have specified at least one "Reviewer", and if necessary, additional reviewers

### CODE REVIEW

**Review by MothOnMars — CHANGES_REQUESTED** [review]
Overall these changes look good, but it occurs to me that we should tweak the order of the bullet points. The "Automated checks pass" bullet should come last, as any changes prompted by the other bullets might cause tests to fail. We may as well also group the two formatting bullets, preceded by the squashing step, i.e.:
```
- [ ] You have squashed your commits into a single commit (exceptions: your PR includes commits with formatting-only changes, such as required by Rubocop or Cookstyle, or if this is a feature branch that includes multiple commits)

- [ ] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket

- [ ] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
```
And per our chat, let's discuss w/ the team how we want to handle "N/A" bullet points.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -25,4 +21,4 @@ Please ensure you have addressed all concerns below before marking a PR "ready f
  
 #### Process Checks
 
-- [ ] You have specified an "Assignee", and if necessary, additional reviewers
\ No newline at end of file
+- [ ] You have specified at least one "Reviewer", and if necessary, additional reviewers
```
[the remark this answers is not in the document]
Comment (amanifarooque) [reply .github/pull_request_template.md]: Updated

**Comment by amanifarooque** [comment]
Made the desired changes & description for handling the "n/a" case - requesting re-review.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -3,26 +3,22 @@
 - Any additional information or context which may help the reviewer
  
 ### Checklist
-Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:
+Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji.
```
Comment (MothOnMars) [inline .github/pull_request_template.md:6]: Oy - I was about to approve this, but realized that we should require an explanation in addition to just the ⚠️ . So something like:
"...replace the checkbox with the :warning: emoji and explain why the step was not completed."

**Review by MothOnMars — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (amanifarooque) [reply .github/pull_request_template.md]: updated

**Comment by amanifarooque** [comment]
@MothOnMars - might as well correct now, apologies for another review, but once this is set I'll copy over the changes to the rest of the repos.

**Pull request merged**